cmd/info/info.go

```diff
@@ -57,6 +61,25 @@ func RunCmd(cmd *cobra.Command) error {
 	}
 	fmt.Println(table)
 
+	fmt.Println()
```

Comment:
Would it be a good idea to also have a "version" subcommand for the CLI? i.e. `ocne version`

Reply:
Would this information be taken out of the info command or put in there, or would it be in both commands?

Reply:
Since we will make it clear in the info command description that the info command has the version information, I will mark this conversation as resolved.